src/Widgets/ButtonWidget/ButtonWidgetComponent.js

```diff
@@ -17,21 +17,19 @@ const ButtonWidgetComponent = Scrivito.connect(({ widget }) => {
   classNames.push(widget.get("style") || "btn-primary");
 
   return (
-    <Scrivito.LinkTag to={target} className={classNames.join(" ")}>
-      {text}
-      <i className="fa fa-angle-right fa-4" aria-hidden="true" />
-    </Scrivito.LinkTag>
+    <AlignmentWrapper alignment={widget.get("alignment")}>
+      <Scrivito.LinkTag to={target} className={classNames.join(" ")}>
+        {text}
+        <i className="fa fa-angle-right fa-4" aria-hidden="true" />
+      </Scrivito.LinkTag>
+    </AlignmentWrapper>
   );
 });
 
-Scrivito.provideComponent("ButtonWidget", ({ widget }) => {
-  if (["center", "right"].includes(widget.get("alignment"))) {
-    return (
-      <div className={`text-${widget.get("alignment")}`}>
-        <ButtonWidgetComponent widget={widget} />
-      </div>
-    );
-  }
-
-  return <ButtonWidgetComponent widget={widget} />;
-});
+function AlignmentWrapper({ alignment, children }) {
```

Comment:
* [ ] FYI Looking at the next commit it becomes clear that the `alignment` setting is not exclusively evaluated here, which makes the naming slightly off. How about `AlignedText`, `TextAlignment`, `TextWrapper` or so?